go.mod

```diff
@@ -19,7 +19,8 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/kcp-dev/apimachinery/v2 v2.0.0-alpha.0
 	github.com/kcp-dev/client-go v0.0.0-20230126185145-aeff170a288b
-	github.com/kcp-dev/kcp/pkg/apis v0.0.0-00010101000000-000000000000
+	github.com/kcp-dev/kcp/pkg/apis v0.11.0
```

Comment:
Revert to 0s

Reply:
So this is a bit interesting. You might know the right way to do this.
`pkg/client` now pulls in `github.com/kcp-dev/kcp/pkg/apis v0.11.0` as it is for APIs. And as it is a child module, I don't think we can do `replaces api ../apis` in it?

So as we have `pkg/client` in root go mod as `./pkg/client` go mod automatically places this version here (it still overrides bellow), but I'm not sure how to force 0s here now.

Reply:
Ah, so it's coming in automatically. That's fine.